src/views/AddTimersView.tsx

```diff
@@ -0,0 +1,133 @@
+import { useContext } from 'react';
+import Countdown from '../components/timers/Countdown';
+import Stopwatch from '../components/timers/Stopwatch';
+import Tabata from '../components/timers/Tabata';
+import XY from '../components/timers/XY';
+import type { TimerProps } from '../utils/TimerProps';
+import { DisplayRepsForText, DisplayTimeForText } from '../utils/helpers';
+import { Button, Buttons, Input, Inputs, MainText, SupportText, Timers } from '../utils/styles';
+import { TimersContext } from './TimerProvider';
+
+const AddTimersView = () => {
+    const { addTimer, timersArray, timerInputs, handleInputChange, removeLastTimer, removeAllTimers, totalQueueSeconds, totalSecondsPassed, currentTimerIndex } = useContext(TimersContext);
+
+    type TimerTitle = 'Stopwatch' | 'Countdown' | 'XY' | 'Tabata';
+
+    const timers: { title: TimerTitle; C: React.ComponentType<TimerProps> }[] = [
+        { title: 'Stopwatch', C: Stopwatch },
+        { title: 'Countdown', C: Countdown },
+        { title: 'XY', C: XY },
+        { title: 'Tabata', C: Tabata },
+    ];
+
+    return (
+        <div style={{ textAlign: 'center', paddingTop: '4rem' }}>
+            Timers available for your custom workout. Enter the time and add to workout.
+            <Timers>
+                {timers.map(timer => (
+                    <div key={`timer-${timer.title}`}>
+                        {timer.title}
+                        <Inputs>
+                            <Input>
+                                <input
+                                    style={{ maxWidth: '4rem', fontSize: '0.75rem', textAlign: 'right' }}
+                                    id="timeMinInput"
+                                    placeholder="Min"
+                                    value={timerInputs[timer.title]?.timeMinInput || ''}
+                                    onChange={e => {
+                                        handleInputChange(timer.title, 'timeMinInput', e.target.value);
+                                    }}
+                                />
+                                :
+                                <input
+                                    style={{ maxWidth: '4rem', fontSize: '0.75rem', textAlign: 'left' }}
+                                    id="timeSecInput"
+                                    value={timerInputs[timer.title]?.timeSecInput || ''}
+                                    placeholder="Sec"
+                                    onChange={e => {
+                                        handleInputChange(timer.title, 'timeSecInput', e.target.value);
+                                    }}
+                                />
+                            </Input>
+                            {(timer.title === 'XY' || timer.title === 'Tabata') && (
+                                <Input>
+                                    <input
```

Comment:
a better pattern here would be to create an `Input` component that encapsulates most of this repeated logic